### PR TITLE
Bf merge react rails

### PR DESCRIPTION
**Pull request by bfcoder**
Fix lint violations

### CODE REVIEW

File: spec/controllers/concerns/iframe_support_spec.rb [hunk 1]
```diff
@@ -1,4 +1,7 @@
 require "rails_helper"
+USER_AGENT =
```
Comment (bfcoder) [inline spec/controllers/concerns/iframe_support_spec.rb]: Moved this huge text up to here

File: spec/lib/url_helper_spec.rb [hunk 2]
```diff
@@ -79,15 +79,16 @@
...
-      expect(UrlHelper.scheme_host_port("https://www.example.com:1244")).to eq("https://www.example.com:1244")
-      expect(UrlHelper.scheme_host_port("https://www.example.com:1234/some/path")).to eq("https://www.example.com:1234")
-      expect(UrlHelper.scheme_host_port("https://www.example.com:1234?some=thing")).to eq("https://www.example.com:1234")
-      expect(UrlHelper.scheme_host_port("http://www.example.com:1234")).to eq("http://www.example.com:1234")
-      expect(UrlHelper.scheme_host_port("http://www.example.com:1234/some/path")).to eq("http://www.example.com:1234")
-      expect(UrlHelper.scheme_host_port("http://www.example.com:1234?some=thing")).to eq("http://www.example.com:1234")
-      expect(UrlHelper.scheme_host_port("www.example.com:1234")).to eq("http://www.example.com:1234")
-      expect(UrlHelper.scheme_host_port("www.example.com:1234/some/path")).to eq("http://www.example.com:1234")
-      expect(UrlHelper.scheme_host_port("www.example.com:1234?some=thing")).to eq("http://www.example.com:1234")
+      example = "www.example.com"
```
Comment (bfcoder) [inline spec/lib/url_helper_spec.rb]: Reuse text in a variable

File: spec/lib/url_helper_spec.rb [hunk 3]
```diff
@@ -99,15 +100,16 @@
...
-      expect(UrlHelper.scheme_host_port("https://foo.example.com:1234")).to eq("https://foo.example.com:1234")
-      expect(UrlHelper.scheme_host_port("https://foo.example.com:1234/some/path")).to eq("https://foo.example.com:1234")
-      expect(UrlHelper.scheme_host_port("https://foo.example.com:1234?some=thing")).to eq("https://foo.example.com:1234")
-      expect(UrlHelper.scheme_host_port("http://foo.example.com:1234")).to eq("http://foo.example.com:1234")
-      expect(UrlHelper.scheme_host_port("http://foo.example.com:1234/some/path")).to eq("http://foo.example.com:1234")
-      expect(UrlHelper.scheme_host_port("http://foo.example.com:1234?some=thing")).to eq("http://foo.example.com:1234")
-      expect(UrlHelper.scheme_host_port("foo.example.com:1234")).to eq("http://foo.example.com:1234")
-      expect(UrlHelper.scheme_host_port("foo.example.com:1234/some/path")).to eq("http://foo.example.com:1234")
-      expect(UrlHelper.scheme_host_port("foo.example.com:1234?some=thing")).to eq("http://foo.example.com:1234")
+      example = "foo.example.com"
```
Comment (bfcoder) [inline spec/lib/url_helper_spec.rb]: Reuse text in a variable

File: README.md [hunk 4]
```diff
@@ -7,49 +7,27 @@ utilize React.
...
-```
-
-## Setting up LTI Starter App
+## <a name="setup"></a>Setting up LTI Starter App
 -----------
 
 ### File Modifications
+-----------
 
-#### Change bin/bootstrap
```
Comment (jbasdf) [inline README.md]: Let's add this back in. It's only relevant for Atomic Jolt team members, but it's important information.

**Pull request merged**